tools/apiview/emitters/cadl-apiview/src/apiview.ts

```diff
@@ -276,10 +274,21 @@ export class ApiView {
   }
 
   stringLiteral(value: string) {
-    this.tokens.push({
-      Kind: ApiViewTokenKind.StringLiteral,
-      Value: `\u0022${value}\u0022`,
-    });
+    const lines = value.split("\n");
+    if (lines.length == 1) {
+      this.tokens.push({
+        Kind: ApiViewTokenKind.StringLiteral,
+        Value: `\u0022${value}\u0022`,
```

Comment:
Do you really need the escape-of-quote here?

Reply:
Yes. Otherwise it won't render with quotes.